### ISSUE TITLE
Rename track

### ISSUE THREAD

**Issue by iHiD** [exercism/reasonml#162]
@ErikSchierboom One to do during ETL week. https://github.com/exercism/reasonml/pull/135#issuecomment-808886131

**Comment by FranklinChen** [comment]
Is this going to happen?

**Comment by iHiD** [comment]
Yep - it's just got lost. The easiest thing is to PR the config.json and docs with relevant updates, then we'll update the slug when we merge. @FranklinChen Happy to do that?

**Comment by BNAndras** [comment]
Are we still planning on updating the slug to `rescript`?

**Comment by BNAndras** [comment]
Given a ReScript track is being independently developed at https://github.com/exercism/rescript, I believe this issue can be closed. There's no intention on my part for it to replace the current ReasonML track since students may still have use for both tracks.
